docs/02-guides/server-side-events/create-an-event.mdx

```diff
@@ -7,9 +7,119 @@ sidebar_position: 1
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+## What is an event emitter
 
-- Create an event emitter
-- Emit an event from your code
+The events are dispatched with what we call Event Emitter. The role of the event emitter is to push events to the Server event pipeline to be processed by the server.
```

Comment:
> what we call Event Emitter

**issue(non-blocking):** Event emitters are a standard concept in many languages, we didn't invent anything.

**suggestion:** Rephrase this sentence
```suggestion
The events are dispatched through the use of an event emitter. The role of the event emitter is to push events to the server event pipeline to be processed by the server.
```

Reply:
updated with your suggestion